tests/test_unhandled_revisions.py

```
@@ -626,22 +634,27 @@ def test_every_unhandled_tag_has_a_fixture():
+def test_change_record_tags_cover_the_whole_property_change_family():
     """CHANGE_RECORD_TAGS repeats the property-change family — pin the overlap.
 
-    A property-change type added to UNHANDLED_REVISION_TAGS but not here would
+    A property-change type added to the revision tags but not here would
     silently stop skip_change_records from skipping its recorded subtree.
+    w:pPrChange is handled (resolved by id) *and* a change record (its
+    recorded subtree is historical), so the family spans both sets.
     """
-    assert set(CHANGE_RECORD_TAGS) <= set(UNHANDLED_REVISION_TAGS)
+    assert set(CHANGE_RECORD_TAGS) <= set(ALL_REVISION_TAGS)
+    assert "w:pPrChange" in CHANGE_RECORD_TAGS and "w:pPrChange" in HANDLED_REVISION_TAGS
```

Ruff (0.16.2)

[warning] Assertion should be broken down into multiple parts

Break down assertion into multiple parts

(PT018)